test/rmq/test_communicator.py

```diff
@@ -1,171 +1,189 @@
 # -*- coding: utf-8 -*-
+from __future__ import absolute_import
 from functools import partial
 import shutil
 import tempfile
 import unittest
 import uuid
+import asyncio
 import shortuuid
 
+import pytest
 from kiwipy import rmq
-from tornado import testing, ioloop
+from six.moves import range
 
 import plumpy
 from plumpy import communications, process_comms
-from test import test_utils
-from test.utils import AsyncTestCase
+from .. import utils
 
 try:
-    import pika
+    import aio_pika
 except ImportError:
-    pika = None
-
-AWAIT_TIMEOUT = testing.get_async_test_timeout()
+    aio_pika = None
```

Comment:
Isn't `aio_pika` and `install_requires` so it should be guaranteed to be installed?

Reply:
I think this was just a mistake in the older versions. I think `plumpy` always required `kiwipy[rmq]` so these guards are not necessary. Let's just remove them until we decide that they really become optional